packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 54.2.0 - February 24, 2022
+
+## Added
+
+#### salesforcedx-vscode
+
+- Run the new *Start Function in Container* command to run a Salesforce Function in a Docker container.  The old command, *Start Function* now runs a function locally. ([PR #3838](https://github.com/forcedotcom/salesforcedx-vscode/pull/3838), [PR #3856](https://github.com/forcedotcom/salesforcedx-vscode/pull/3856))
+
+## Fixed
+
+#### salesforcedx-vscode
+
+- We resolved the issue with the *Diff Folder Against Org* command being visible from command palette even when an SFDX project wasn’t open. ([PR #3843](https://github.com/forcedotcom/salesforcedx-vscode/pull/3843)).
+
+#### salesforcedx-vscode-apex
+
+- We removed some unessential apex classes from the Apex LSP jar file. ([PR #3844](https://github.com/forcedotcom/salesforcedx-vscode/pull/3844))
```

Comment:
```suggestion
- We removed some unessential Apex classes from the Apex LSP .jar file. ([PR #3844](https://github.com/forcedotcom/salesforcedx-vscode/pull/3844))
```

Reply:
".jar" is what CCX recommends, although honestly I don't always follow the rule.  I think it looks funky...

https://confluence.internal.salesforce.com/display/DOCTEAM/file+names+and+extensions